packages/core/src/services/ai/index.ts

```diff
@@ -121,8 +121,8 @@ export async function ai({
       messages,
       provider,
       apiKey,
+      url: url ?? undefined,
```

Comment:
This is not the same as before. Make sure passing undefined for url is not having side effects

Reply:
Why it is not the same? In the function that parameter is optional

Reply:
Now is always passing `url: undefined` before was passing nothing. Probably does not matter at all. It depneds on how is used url under the hood

Reply:
I understand that but I am calling a function with an optional parameter, it should be undefined in both cases